content/ui-patterns/degraded-experiences.mdx

```diff
@@ -193,6 +131,68 @@ If you're not using a component that supports error states, replace the content
   src="https://github.com/primer/design/assets/2313998/bde4367e-8881-41f5-9d06-a2ea5cd79fd6"
 />
 
+### Removing UI
+
+Be careful about choosing to remove affected UI without explanation. This could cause users to be confused about whether that part of the UI is hidden due to an error, or if it's been permanently removed.
```

Comment:
```suggestion
Be mindful when choosing to remove affected UI without explanation, as this could cause confusion for users about whether that part of the UI is hidden due to an error, or if it's been permanently removed.
```